docs/CodeDevelopment/prerequisites.md

```diff
@@ -34,44 +34,43 @@ the expectation is that between Project Mu and TianoCore Edk2 you could use any
     2. Native Channel support (this will help in corp environments)
     3. Check the box to "Enable Git Credential Manager"
 
-### Visual Studio 2019 **preferred**
+### Visual Studio 2022 **preferred**
 
 1. Download latest version of VS build Tools to c:\TEMP
 
     ``` cmd
-    https://aka.ms/vs/16/release/vs_buildtools.exe
+    https://aka.ms/vs/17/release/vs_buildtools.exe
     ```
 
 2. Install from cmd line with required features (this set will change over time).
 
     ``` cmd
     C:\TEMP\vs_buildtools.exe --quiet --wait --norestart --nocache --installPath C:\BuildTools ^
     --add Microsoft.VisualStudio.Component.VC.CoreBuildTools --add Microsoft.VisualStudio.Component.VC.Tools.x86.x64 ^
-    --add Microsoft.VisualStudio.Component.Windows10SDK.17763 --add Microsoft.VisualStudio.Component.VC.Tools.ARM ^
+    --add Microsoft.VisualStudio.Component.Windows11SDK.22000 --add Microsoft.VisualStudio.Component.VC.Tools.ARM ^
     --add Microsoft.VisualStudio.Component.VC.Tools.ARM64
     ```
 
-See component list here for more options. <https://docs.microsoft.com/en-us/visualstudio/install/workload-component-id-vs-build-tools?view=vs-2019>
+See component list here for more options. <https://docs.microsoft.com/en-us/visualstudio/install/workload-component-id-vs-build-tools?view=vs-2022>
 
-### Visual Studio 2017
+### Visual Studio 2019
 
 1. Download latest version of VS build Tools to c:\TEMP
 
     ``` cmd
-    https://aka.ms/vs/15/release/vs_buildtools.exe
+    https://aka.ms/vs/16/release/vs_buildtools.exe
     ```
 
 2. Install from cmd line with required features (this set will change over time).
 
     ``` cmd
     C:\TEMP\vs_buildtools.exe --quiet --wait --norestart --nocache --installPath C:\BuildTools ^
     --add Microsoft.VisualStudio.Component.VC.CoreBuildTools --add Microsoft.VisualStudio.Component.VC.Tools.x86.x64 ^
-    --add Microsoft.VisualStudio.Component.Windows10SDK.17763 --add Microsoft.VisualStudio.Component.VC.Tools.ARM ^
+    --add Microsoft.VisualStudio.Component.Windows10SDK.19041 --add Microsoft.VisualStudio.Component.VC.Tools.ARM ^
     --add Microsoft.VisualStudio.Component.VC.Tools.ARM64
     ```
 
-See component list here for more options.
-<https://docs.microsoft.com/en-us/visualstudio/install/workload-component-id-vs-build-tools?view=vs-2017>
+See component list here for more options. <https://docs.microsoft.com/en-us/visualstudio/install/workload-component-id-vs-build-tools?view=vs-2019>
 
 ### Optional - Windows Driver Kit
```

Comment:
Not entirely related: would we stick with the version specified in this section?